.github/workflows/tests.yml

```diff
@@ -225,6 +225,13 @@ jobs:
             fi
           fi
 
+          cat <<EOT >> mplsetup.cfg
+          [rc_options]
+          ##backend=Agg
```

Comment:
Does this do anything with the double comment?

Reply:
The goal was to not change the resultant `matplotlibrc` file from what it would be without this configuration.  Without the bug fix this would have put the webagg.port on the same line so it would be missing and any import would fail..

Reply:
But the substitution doesn't care that this is double commented, it only inserts the value from here with a fixed key. And since this file is parsed by `ConfigParser`, it's not going to see this line at all, anyway.

Reply:
🤦🏻 🐑 I understand now.

Reply:
Third times the charm?